spec/parser/cfn_parser_ec2_instance_spec.rb

```diff
@@ -29,14 +25,23 @@
     it 'returns ec2 instance with security group' do
       yaml_test_templates('ec2_instance/instance_with_sgid_list_ref').each do |test_template|
         cfn_model = @cfn_parser.parse IO.read(test_template)
-
```

Comment:
while the spacing/grouping might not always be consistent, the whitespace is meant to enhance readability.  i'd advise keep 32, drop 34, add one after 35 and drop 37.  parse, code supporting assertion1, code supporting assertion2